<div class="col-12 about__content">
    <h3 class="text-muted">About KEA</h3>

    <p>Protein kinases are enzymes that catalyze the transfer of a phosphate group from ATP to another protein on a
        threonine, a serine, or a tyrosine residue. The reversible addition of a phosphate group to a protein can
        profoundly affect its reversible activity, stability, localization, and interactions with other molecules. Of
        the hundreds of thousands of potential phosphorylation sites, each protein kinase recognizes between one to a
        few hundred substrates. Thus, kinases serve a critical and central role in regulating essentially all cellular
        processes, and their aberrant activity is recognized as a cause of many human diseases. Phosphoproteomic
        signatures, which represent the differential phosphorylation state of cellular proteins between two cellular
        states, can provide a snapshot of the intracellular signaling networks that are differentially active between
        two conditions, for instance, between diseased and healthy cells. The enrichment of known kinase substrates in a
        set of differentially phosphorylated proteins can serve as a marker of the activity of the upstream kinases,
        thus providing insights into physiological and pathophysiological mechanisms. Kinase Enrichment Analysis 3
        (KEA3) infers upstream kinases whose putative substrates are overrepresented in a user-inputted list of genes or
        differentially phosphorylated proteins. The KEA3 database contains putative kinase-substrate interactions
        collected from publicly available datasets. Gene sets of putative kinase substrates are used as the primary
        units of analysis in KEA3. These gene sets are organized in gene set "libraries." Libraries are supersets of
        kinase substrate sets that are aggregated based on the database from which they are derived.</p>

    <p><strong>Library Construction</strong></p>
    <p> All protein and gene symbols were mapped to 2019 HGNC-approved gene symbols using the custom R package <a
            href="https://github.com/MaayanLab/genesetr">genesetr</a>. Gene or protein symbols that could not be mapped
        using synonym or alias matching were discarded. The set of 520 unique <a href="https://www.genenames.org/">HGNC-mappable</a>
        human protein kinases used for KEA3 are derived from <a href="https://www.ncbi.nlm.nih.gov/pubmed/12471243">Manning
            et al.</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/17557329">Miranda-Saavedra and Barton</a>, and the
        <a href="https://druggablegenome.net/">Illuminating the Druggable Genome</a> (IDG) project.
    </p>

    <p><i>Kinase-substrate interactions</i></p>
    <p>Kinase-substrated interactions were collected from <a
            href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/#SD5" target="_blank">Cheng <i>et al.</i></a>, <a
            href="https://omictools.com/phosd-tool">PhosD</a>, and <a
            href="http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/" target="_blank">PTMsigDB</a> to generate
        the <i>Cheng.KSIN</i>, <i>PhosD.All</i>, and <i>PTMsigDB</i> KEA3 libraries, respectively. The <a
                href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/#SD5" target="_blank">Cheng <i>et al.</i></a>kinase-substrate
        interaction network (KSIN) dataset incorporates human kinase-substrate interactions from the <a href="">Phospho.ELM</a>,<a
                href="https://genome.cshlp.org/content/13/10/2363.full.html">HPRD</a>, <a
                href="https://phosphonetworks.org/">PhosphoNetworks</a>, and<a
                href="https://www.phosphosite.org/homeAction.action">PhosphoSitePlus</a> databases.<a
                href="http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/" target="_blank">PTMsigDB</a>contains a
        collection of kinase phosphorylation signatures derived from <a
                href="https://www.phosphosite.org/homeAction.action">PhosphoSitePlus</a>.<a
                href="https://omictools.com/phosd-tool">PhosD</a> predicts kinase-substrate interactions based on
        protein domains.</p>


    <p><i>Protein-protein interactions </i></p>
    <p>Kinase-interacting proteins were collected from the PPI databases <a
            href="https://thebiogrid.org/">BioGRID</a>, <a href="https://mentha.uniroma2.it/">mentha</a>, <a
            href="https://mint.bio.uniroma2.it/">MINT</a>, <a href="http://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie/">HIPPIE</a>,
        <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/">Cheng <i>et al.</i></a>, <a
                href="http://honig.c2b2.columbia.edu/preppi">prePPI</a>, and <a href="https://string-db.org/">STRING</a>
        to
        generate the <i>BioGRID</i>, <i>mentha</i>, <i>MINT</i>, <i>HIPPIE</i>, <i>Cheng.bind</i>, <i>prePPI</i>, and
        <i>STRING.bind</i>
        KEA3 libraries, respectively. The <a href="https://thebiogrid.org/">BioGRID</a> and <a
                href="https://mint.bio.uniroma2.it/">MINT</a>databases contain PPIs from high- and low-throughput
        experiments that were manually curated from the literature. <a href="https://mentha.uniroma2.it/">Mentha</a> is
        a PPI network (PPIN) that contains molecular interactions aggregated and updated weekly from <a
                href="https://mint.bio.uniroma2.it/">MINT</a>, <a href="https://www.ebi.ac.uk/intact/">IntAct</a>, <a
                href="https://thebiogrid.org/">BioGRID</a>, <a href="http://matrixdb.univ-lyon1.fr/">MatrixDB</a>, and
        the<a href="https://dip.doe-mbi.ucla.edu/dip/Main.cgi">Database of Interacting Proteins (DIP)</a>.<a
                href="http://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie/">HIPPIE</a>aggregates experimentally determined
        PPIs from <a href="https://www.ebi.ac.uk/intact/">IntAct</a>, <a href="https://mint.bio.uniroma2.it/">MINT</a>,
        <a href="https://thebiogrid.org/">BioGRID</a>, <a href="https://genome.cshlp.org/content/13/10/2363.full.html">HPRD</a>,
        <a href="https://dip.doe-mbi.ucla.edu/dip/Main.cgi">DIP</a>, <a
                href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC165503/">BIND</a>, and <a
                href="http://mips.helmholtz-muenchen.de/proj/ppi/">MMPI</a>.<a
                href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/">Cheng <i>et al.</i></a> used PPIs collected
        from<a href="https://www.ebi.ac.uk/intact/">IntAct</a>, <a href="https://mint.bio.uniroma2.it/">MINT</a>, <a
                href="https://thebiogrid.org/">BioGRID</a>, <a href="https://dip.doe-mbi.ucla.edu/dip/Main.cgi">DIP</a>,
        <a
                href="https://genome.cshlp.org/content/13/10/2363.full.html">HPRD</a>, and <a
                href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1347366/">MIPS MPact</a>.<a
                href="http://honig.c2b2.columbia.edu/preppi">PrePPI</a> consists of predicted PPIs that were determined
        using a Bayesian framework that relies on three-dimensional structural, functional evolutionary, and expression
        information to make PPI predictions. Note there is overlap and redundancy among the databases that aggregate
    </p>

    <p><i>Other</i></p>
    <p>All kinase interactions in the <a href="https://string-db.org" target="_blank">STRING</a> database were used to
        generate the <i>STRING</i> KEA3 library. <a href="https://string-db.org" target="_blank">STRING</a>includes
        physical interaction, co-expression, co-occurrence in the literature, and evolutionary co-occurrence, among
        other association types.
    </p>

    <p><strong>Kinase Substrate Over-representation Analysis</strong> - The goal of KEA3 is to predict kinases
        associated with user-input sets of genes/proteins. Discrete query gene/protein sets are compared to <a
                href="#download">KEA3 libraries</a> of putative kinase substrate sets assembled from multiple publically
        available datasets. The Fisher's Exact Test, with a background size of 20,000, is used to compare the input
        gene/protein set to the kinase substrate sets in order to determine which kinases may be most closely associated
        with the input gene/protein set.
    </p>
    <p><strong>Input</strong> - Users can submit a set of human gene or protein symbols for kinase enrichment analysis.
        Commonly, the gene set may be a set of differentially expressed genes, or a set of differentially phosphorylated
        proteins with the goal of forming hypotheses about which upstream kinases might be responsible for the changes
        in the phenotype.
    </p>

    <p><strong>Kinase Co-expression Network Visualizations</strong> - The KEA3 application displays a scatter plot of
        points representing human kinases based on their co-expression similarity. This kinase coexpression network was
        constructed by conducting a <a
                href="https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/">Weighted Gene
            Co-expression Network Analysis (WGCNA)</a> over human kinase expression data. WGCNs were constructed from <a
                href="https://gtexportal.org/home/">GTEx</a> samples, <a href="https://portal.gdc.cancer.gov">TCGA</a>
        samples, and<a href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a> samples. The network layouts were determined
        using Allegro Fruchterman-Reingold Clustering in <a href="https://cytoscape.org">Cytoscape</a>. For the <a
                href="https://gtexportal.org/home/">GTEx</a> networks, WGCNA kinase module summary expression profiles,
        also known as Eigen-genes, were correlated with gene expression profiles from all <a
                href="https://gtexportal.org/home/">GTEx</a> tissue samples to determine strong module-tissue
        associations. Each <a href="https://gtexportal.org/home/">GTEx</a> WGCN module may be colored by its most highly
        correlated tissue. Tissues are aggregated into more general tissue types (e.g. brain) and more specific tissue
        types (e.g. cerebellum, frontal cortex):</p>
    <div class="table-responsive">
        <table class="table table-sm" id="gtextable">
            <thead>
            <tr>
                <th scope="col">General Tissue</th>
                <th scope="col">Specific Tissue</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>Adipose Tissue</td>
                <td>Subcutaneous</td>
            </tr>
            <tr>
                <td>Adrenal Gland</td>
                <td>Adrenal Gland</td>
            </tr>
            <tr>
                <td rowspan="2" style="vertical-align: middle;">Blood</td>
                <td>Whole Blood</td>
            </tr>
            <tr>
                <td>EBV-transformed lymphocytes</td>
            </tr>
            <tr>
                <td>Blood Vessel</td>
                <td>Artery - Tibial</td>
            </tr>
            <tr>
                <td rowspan="3" style="vertical-align: middle;">Brain</td>
                <td>Cerebellum</td>
            </tr>
            <tr>
                <td>Frontal Cortex (BA9)</td>
            </tr>
            <tr>
                <td>Spinal cord (cervical c-1)</td>
            </tr>
            <tr>
                <td rowspan="2" style="vertical-align: middle;">Colon</td>
                <td>Sigmoid</td>
            </tr>
            <tr>
                <td>Transverse</td>
            </tr>
            <tr>
                <td rowspan="2" style="vertical-align: middle;">Esophagus</td>
                <td>Muscularis</td>
            </tr>
            <tr>
                <td>Mucosa</td>
            </tr>
            <tr>
                <td>Heart</td>
                <td>Atrial Appendage</td>
            </tr>
            <tr>
                <td>Liver</td>
                <td>Liver</td>
            </tr>
            <tr>
                <td>Muscle</td>
                <td>Skeletal</td>
            </tr>
            <tr>
                <td>Nerve</td>
                <td>Tibial</td>
            </tr>
            <tr>
                <td>Pancreas</td>
                <td>Pancreas</td>
            </tr>
            <tr>
                <td>Pituitary</td>
                <td>Pituitary</td>
            </tr>
            <tr>
                <td>Prostate</td>
                <td>Prostate</td>
            </tr>
            <tr>
                <td rowspan="2" style="vertical-align: middle;">Skin</td>
                <td>Sun Exposed (Lower leg)</td>
            </tr>
            <tr>
                <td>Transformed fibroblasts</td>
            </tr>
            <tr>
                <td>Testis</td>
                <td>Testis</td>
            </tr>
            <tr>
                <td>Thyroid</td>
                <td>Thyroid</td>
            </tr>
            <tr>
                <td>Uterus</td>
                <td>Uterus</td>
            </tr>
            </tbody>
        </table>
    </div>
    <p><a href="https://portal.gdc.cancer.gov">TCGA</a> module Eigen-genes were correlated with tumor gene expression
        profiles to determine strong module-tumor type associations. Each <a
                href="https://portal.gdc.cancer.gov">TCGA</a> WGCN module may be colored by its most highly correlated
        tumor type. Finally, the <a href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a> module Eigen-genes were
        correlated with <a href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a> tissue types. The <a
                href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a> network modules may be colored by the most highly
        correlated tissue type.</p>
    <h3 class="text-muted">Tutorial</h3>
    <ol>
        <li><strong>Navigating Your Results</strong> - Results tables from the integration methods MeanRank
            and TopRank
            and from each KEA3 kinase substrate library are accessible from a dropdown menu.
            The MeanRank table is by default the first table shown because it performed the best across the
            KEA3 benchmarks. Tables may
            be searched by entering terms in the search box, sorted by column by clicking on the
            column header, and copied to the clipboard by clicking on the Copy button. Results
            for the KEA3 kinase substrate libraries are sorted by the Fisher's Exact Test p-value.
            This provides a ranking of kinases whose putative substrates
            are most closely similar to the query set. Integrated results, which take into account
            results from all libraries, are sorted in ascending order by score. Lower scores indicate
            more relevancy to the kinase substrate set. The top 100 results are returned for each table. The
            full kinase ranking and results
            for each integration method and library may be downloaded as a tab-separated (tsv) file using a
            link found at the footer of each
            results table. Full results are also available via the
            <a href="#api">KEA3 API</a>.
            <img src="static/tables.gif" class="img-fluid"></li>
        <li><strong>Visualizing Your Results</strong> - A global kinase co-expression
            network, a local results-specific co-regulatory network, bar charts, and a clustergram are
            available to aid in interpreting the results.
            <ul>
                <li><strong>Kinase Co-expression Networks</strong></li>

                The kinase co-expression networks are provided to help users visualize their top ranking
                kinases
                in the context of the larger human kinase phosphorylation network. There are three networks
                generated from GTEx,
                TCGA, and ARCHS4 expression data. Users may toggle between these three networks using the
                dropdown menu.
                <img src="static/coexp.gif" class="img-fluid">

                The slider above the network designates the number of kinases to highlight in the network.
                The kinases that are highlighted are the top results
                from the library that is selected from library selection dropdown menu.
                Network node coloring options provide additional information about the tissues or tumor
                types the kinases may be
                most active in. The network may be zoomed, panned, and saved as an SVG image. There are
                other options, for
                example, hiding the legend or switching to full screen. These options can be accessed by
                clicking
                the "Network Options" hamburger.
                </li>

                <li><strong>Kinase Co-regulatory Networks</strong></li>
                Kinase-kinase phosphorylation networks are dynamically generated using the top results of
                the selected library. Edges between kinases are defined
                by evidence from the KEA3 libraries and are directed where evidence from a KSI library
                supports the interaction and undirected where evidence from a PPI library supports the
                interaction.
                The supporting evidence for each
                edge is summarized in a tooltip that appears upon hovering over each edge. The slider above
                the network indicates the number of top kinase results
                to include in the network from the library results indicated in the library selection
                dropdown.
                <img src="static/coreg.gif" class="img-fluid">
                <li><strong>Bar Charts</strong></li>
                Bar charts corresponding to the top kinases of the selected library display the
                -log10(pvalue) for KEA3 kinase substrate library results, and an integrated
                rank score for the integrated library results. The numer of top kinases on the y-axis may be
                modifed using the slider.
                <img src="static/bar.gif" class="img-fluid">

                <li>Clustergram</li>
                The interactive clustergram shows the overlapping query substrate targets among top library
                results.
                <img src="static/cluster.gif" class="img-fluid">
            </ul>
    </ol>
</div>