<!DOCTYPE html>
<html>

<head>
	<!-- Site made with Mobirise Website Builder v4.8.1, https://mobirise.com -->
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
	<link rel="shortcut icon" href="assets/images/KEA3_logo_transparent.png" type="image/x-icon">
	<meta name="description" content="KEA3 Web Tool">
	<title>KEA3</title>
	<link rel="stylesheet" href="assets/web/assets/mobirise-icons/mobirise-icons.css">
	<link rel="stylesheet" href="assets/tether/tether.min.css">
	<link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
	<link rel="stylesheet" href="assets/bootstrap/css/bootstrap-grid.min.css">
	<link rel="stylesheet" href="assets/bootstrap/css/bootstrap-reboot.min.css">
	<link rel="stylesheet" href="assets/socicon/css/styles.css">
	<link rel="stylesheet" href="assets/dropdown/css/style.css">
	<link rel="stylesheet" href="assets/theme/css/style.css">
	<link rel="stylesheet" href="assets/mobirise/css/mbr-additional.css" type="text/css">
	<link rel="stylesheet" href="assets/networkd3/network_style.css">
	<!-- <link rel="stylesheet" href="assets/datatables/jquery.dataTables.min.css"> -->
	<!-- <link rel="stylesheet" href="assets/datatables/buttons.dataTables.min.css"> -->
	<link rel="stylesheet" href="assets/datatables/dataTables.bootstrap4.min.css">
	<link rel="stylesheet" href="assets/tooltips/css_tooltips.css">
	<link rel="stylesheet" href="assets/networkd3/network_options.css">
	<link rel="stylesheet" href="assets/spectrum/spectrum.css">
	<link rel="stylesheet" href="assets/chea-query/pretty-headers.css">
	<link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
	<link rel="stylesheet" href="assets/chea-query/result_scroll.css">
	<link rel="stylesheet" href="assets/chea-query/submit-query.css">
	<link rel="stylesheet" href="assets/chea-query/d3barchart.css">
	<link rel="stylesheet" href="assets/networkd3/netprogressbar.css">
	<link rel="stylesheet" href="assets/styles.css">
	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.8.0/Chart.min.css">
	<link rel="stylesheet" type="text/css" href="assets/chea3home/homepage.css">
	<link rel="stylesheet" type="text/css" href="assets/chea3home/df_style.css">
	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/css/bootstrap-select.min.css">

</head>

<body style="background:white">

	<!-- Loading screen -->
	<div id="loading-screen" class="mbr-overlay d-none" style="color:#7A1920">
		<div class="c_spinner"></div>
		<div class="ellipsis"></div>
	</div>

	<!-- Navigation Bar -->
	<section class="menu cid-r0JOqR1tf7" id="menu1-0">
		<nav class="navbar navbar-expand beta-menu navbar-dropdown align-items-center navbar-fixed-top navbar-toggleable-sm"
			style="min-height: 25px; padding: 0px; background:#7A1920">
			<div class="container py-1">
				<button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse"
					data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
					aria-label="Toggle navigation">
					<div class="hamburger">
						<span></span> <span></span> <span></span> <span></span>
					</div>
				</button>
				<div class="menu-logo">
					<div class="navbar-brand" style="margin-left: 10px; min-height: 1rem">
						<span class="navbar-logo"> <a href="#top"> <img src="assets/images/KEA3_logo_transparent.png" alt="KEA3"
									title="" style="height: 1.5rem;">
							</a>
						</span> <span class="navbar-caption-wrap"><a class="navbar-caption text-white display-4"
								href="#top" style="padding-right:0;margin-right:0"> KEA3&nbsp
							</a><a class="navbar-caption text-white display-4"
								style="padding-left:0;margin-left:0;font-weight:300" href="#top">- Kinase Enrichment
								Analysis Version 3</a></span>
					</div>
				</div>
				<div class="collapse navbar-collapse" id="navbarSupportedContent"
					style="margin-right: 10px; padding-right: 0px">
					<ul class="navbar-nav nav-dropdown" data-app-modern-menu="true">
						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="index.html#content4-o"><span
									class="mbri-setting mbr-iconfont mbr-iconfont-btn"></span> About</a>
						</li>
							<li class="nav-item"><a class="nav-link link text-white display-4"
								href="index.html#content4-about"><span
									class="mbri-idea mbr-iconfont mbr-iconfont-btn"></span>Tutorial</a></li>
									
						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="index.html#content4-z"><span
									class="mbri-laptop mbr-iconfont mbr-iconfont-btn"></span>API</a></li>
									
						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="https://github.com/MaayanLab/KEA3web" target="_blank"><span
									class="mbri-github mbr-iconfont mbr-iconfont-btn"></span>GitHub</a></li>

						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="index.html#content4-13"><span
									class="mbri-save mbr-iconfont mbr-iconfont-btn"></span>Download</a></li>
					</ul>

					<!-- <a id="hitcount"> <span class="mbr-iconfont mbr-iconfont-btn"></span></a> -->

				</div>
			</div>
		</nav>
	</section>
	<br>


	<!-- Block 1.1: Query Submissions -->
	<section class="mbr-section" id="homepage" style="background:white;padding-bottom:0">
			
			<div class="container" style="background: white">
			<div class="row" style="background: white">
				<div class="col-8 offset-2" style="background:white">
	
					<div id="tfea-submission">
						<h1 class="mbr-section-title mbr-bold mbr-fonts-style display-5 mt-5 mb-0 pt-5 pb-0">
							Submit Your Gene Set for Analysis with KEA3
						</h1>
						<!-- <p class="mbr-text mbr-fonts-style display-7"
											style="margin-bottom: 0; padding-bottom: 2rem">Discover
											transcription factors associated with sets of target genes</p> -->
						<div class="form-container" style="padding: 0; border: ''">
							<div class="media-container-column" data-form-type="formoid">
					
								<form class="mbr-form" action="">
									<div class="form-group" data-for="message">
										<button type="button" class="btn btn-link display-7 px-0 py-2" style="color:#7A1920;" id="example-genelist"
											type="submit">Example protein list</button>
										<div class="d-inline-block mx-2">or</div>
										<input id="file-input" type="file" style="width: 201px;" accept=".txt">
										<span class="mbri-question ml-2" data-container="body" data-toggle="popover" data-placement="right"
											data-trigger="hover" data-title="<div class='px-1 py-2'>What file types can be uploaded?</div>"
											data-content="<div style = 'max-width:400px' class='px-3 py-2'>KEA3 supports uploading gene lists as text files containing one gene per line.</div>"
											data-html="true"></span>
										<textarea class="form-control 3px pt-2" rows="7" placeholder="Submit protein list with one gene per row."
											id="genelist" cols=""></textarea>
					
									</div>
					
									<button type="button" class="btn btn-primary" id="submit-genelist" type="submit"
										style="background-color:#7A1920;padding: .5;">Submit</button>
									<br>
									<span class="display-7" id="genecheck" style="font-size:70%">0 symbols entered, 0
										duplicates, 0 valid symbols</span>
										<span class="mbri-question ml-2" data-container="body" data-toggle="popover" data-placement="right" data-trigger="hover" data-title="<div class='px-1 py-2'>What gene symbols are accepted?</div>" data-content="<div style = 'max-width:400px' class='px-3 py-2'>KEA3 supports <a target='_blank' href ='https://www.genenames.org/'>HGNC-approved</a> gene symbols. Symbols that are not included in this set are not analyzed.</div>" data-html="true"></span>
	
								</form>
					
							</div>
						</div>
					</div>
	
				</div>
	
			</div> <!-- end row -->
		</div> <!-- end container -->

	</section>
	
	<!-- Block 1.2: Results Page -->
	<section class="mbr-section d-none" id="results" style="background:white;">
		<div class="container">
			<a class="btn btn-primary float-left position-absolute" id="newquerybutton" href="/kea3" style="padding: .5rem 1.5rem">New Query</a>
			<div class="h1 text-center">Browse KEA3 Results</div>
			<hr width="100%" class="my-4">

			<!-- Selection -->
			<div id="options-row" class="row">

				<!-- Library Selectpicker -->
				<div class="col-3">
					<div>Select a library:<span class="mbri-question ml-2" data-container="body" data-toggle="popover" data-placement="right" data-trigger="hover" data-title="<div class='px-1 py-2'>What are libraries?</div>" data-content="<div style = 'max-width:400px' class='px-3 py-2'>Each library is a collection of putative kinase-substrate interactions that your gene set is queried against. The Integrated Results aggregate kinase predictions from the nine other KEA3 libraries.</div>" data-html="true"></span></div>
					<select id="library-selectpicker" class="cursor-pointer" data-header="Select a library:">
						<optgroup label="Integrated Results">
							<option data-subtext="Average integrated ranks across libraries" value="Integrated--meanRank">Mean Rank</option>
							<option data-subtext="Top integrated rank across libraries" value="Integrated--topRank">Top Rank</option>
						</optgroup>
						<optgroup label="Kinase-Substrate Interaction">
							<option data-subtext="Aggregation of KSIs from public databases" value="ChengKSIN">Cheng KSIN</option>
							<option data-subtext="Experimentally determined KSIs curated from literature" value="PTMsigDB">PTMsigDB</option>
							<option data-subtext="KSI predictions based on protein domain" value="PhosDAll">PhosD All</option>
						</optgroup>
						<optgroup label="Protein-Protein Interaction">
							<option data-subtext="PPI predictions based on structure/evolution/expression" value="prePPI">prePPI</option>
							<option data-subtext="Experimentally determined PPIs curated from literature" value="BioGRID">BioGRID</option>
							<option data-subtext="Aggregation of PPIs from public databases" value="mentha">mentha</option>
							<option data-subtext="Experimentally determined PPIs curated from literature" value="MINT">MINT</option>
							<option data-subtext="Aggregation of PPIs from public databases" value="HIPPIE">HIPPIE</option>
							<option data-subtext="Aggregation of PPIs from public databases" value="STRINGbind">STRING bind</option>
							<option data-subtext="Aggregation of PPIs from public databases" value="ChengPPI">Cheng PPI</option>
						</optgroup>
						<optgroup label="All Associations">
							<option data-subtext="Direct and indirect PPI interactions and associations" value="STRING">STRING</option>
						</optgroup>
					</select>
					</select>
				</div>

				<!-- Slider -->
				<div class="col-4 tf-selector d-none">
					<div>Select kinases to visualize:<span class="mbri-question ml-2" data-container="body" data-toggle="popover" data-placement="right" data-trigger="hover" data-title="<div class='px-1 py-2'>Where are the kinases visualized?</div>" data-content="<div style = 'max-width:400px' class='px-3 py-2'> The top kinases from the selected library are visualized in the Global Network, Local Network, and Bar Chart tabs.</div>" data-html="true"></span></div>
					<div class="font-italic mt-1" style="font-size: small">Top <span id="nr-selected-tfs">10</span> kinases selected</div>
					<input id="tf-slider" class="slider w-100" type="range" min="1" max="50" value="10" style="color:#7A1920">
				</div>
				
				<!-- Colorpicker -->
				<div id="colorpicker-col" class="col-3 tf-selector d-none">
					<div>Select a color:<span class="mbri-question ml-2" data-container="body" data-toggle="popover" data-placement="right" data-trigger="hover" data-title="<div class='px-1 py-2'>Where are the kinases visualized?</div>" data-content="<div style = 'max-width:400px' class='px-3 py-2'>The top kinases from the selected library are visualized in the Global Network, Local Network, and Bar Chart tabs.</div>" data-html="true"></span></div>
					<input type='text' id="colorpicker"/>
				</div>
			</div>




			<!-- Nav -->
			<nav class="pt-3">
				<div class="nav nav-tabs nav-justified" id="nav-tab" role="tablist">
					<a class="nav-item nav-link rounded-0 active" id="nav-tables-tab" data-toggle="tab" href="#nav-tables" role="tab"
						aria-controls="nav-tables" aria-selected="true">Tables</a>
					<a class="nav-item nav-link rounded-0" id="nav-global-network-tab" data-toggle="tab" href="#nav-global-network" role="tab"
						aria-controls="nav-global-network" aria-selected="false">Global Network</a>
					<a class="nav-item nav-link rounded-0" id="nav-local-network-tab" data-toggle="tab" href="#nav-local-network" role="tab"
						aria-controls="nav-local-network" aria-selected="false">Local Network</a>
					<a class="nav-item nav-link rounded-0" id="nav-barchart-tab" data-toggle="tab" href="#nav-barchart" role="tab"
						aria-controls="nav-barchart" aria-selected="false">Bar Chart</a>
					<a class="nav-item nav-link rounded-0" id="nav-clustergram-tab" data-toggle="tab" href="#nav-clustergram" role="tab"
						aria-controls="nav-clustergram" aria-selected="false">Clustergram</a>
				</div>
			</nav>

			<!-- Tab Content -->
			<div class="tab-content" id="nav-tabContent">
				
				<!-- Tables Tab -->
				<div class="tab-pane border-top-0 rounded-bottom px-4 py-3 fade show active" id="nav-tables" role="tabpanel" aria-labelledby="nav-tables-tab">
					<div id="tables-wrapper"></div>
					<div id="results-tables" style="background-color: #333333; min-height: 500px; display: none;"></div>
				</div>
				

				<!-- Global Network Tab -->
				<div class="tab-pane border-top-0 rounded-bottom px-4 py-3 fade" id="nav-global-network" role="tabpanel" aria-labelledby="nav-global-network-tab">

					<!-- Ali Code -->
					<div class="tab-content networkcol" id="pills-tabContent">
						<div class="tab-pane fade show active" id="pills-coexpression" role="tabpanel" aria-labelledby="pills-coexpression-tab">
							<div style="background: white" class="networkcol" id="tfnet">
								<!-- Side network navigation menu -->
								<div id="netoptsidenav" class="sidenav" style="float: top">
									<a href="javascript:void(0)" class="closebtn" onclick="closeNav('netoptsidenav')">&times;</a>
									<a class="nav-link link text-white display-4">
										<span class="mbri-change-style mbr-iconfont"></span>
										Color by:
									</a>
									<div class="form-group" style="width: 80%; padding-left: 20px" id="colorbyoptions">
										<select class="form-control" id="colorby" onchange="recolorAllNodes();setGTExLegendView()">
											<option>Tissue (general)</option>
											<option>Tissue (specific)</option>
											<option>GO Enrichment</option>
											<option>WGCNA modules</option>
											<option>none</option>
										</select>
									</div>
									<a class="nav-link link text-white display-4" id="gtextablelink" href="#gtextable">GTEx tissue table</a>
									<a class="nav-link link text-white display-4" onclick="deleteNetwork();zm=1;drawNetwork();">
										<span class="mbri-drag-n-drop2 mbr-iconfont"> </span>Zoom to fit
									</a>
									<a class="nav-link link text-white display-4" onclick="requestFullScreen('tfnet')">
										<span class="mbri-drag-n-drop mbr-iconfont"></span>
										Fullscreen
									</a>
									<a class="nav-link link text-white display-4" onclick="saveSvg('net_svg','network.svg')">
										<span class="mbri-camera mbr-iconfont"> </span>
										Save SVG
									</a>
									<!-- Show node labels: auto, always, never -->
									<div class="checkbox">
										<a class="nav-link link text-white display-4">
											<span class="mbri-edit2 mbr-iconfont"></span>
											Show node labels:
										</a>
									</div>
									<div class="form-group" style="width: 80%; padding-left: 20px">
										<select class="form-control" id="labelview" onchange="setLabelView()">
											<option>auto</option>
											<option>always</option>
											<option>never</option>
										</select>
									</div>
									<a class="nav-link link text-white display-4">
										<span class="mbr-iconfont">
											<input type="checkbox" id="legend_checkbox" class="checkbox-inline" checked="checked" value="1">
										</span>
										Show Legend
									</a>
								</div>
								<div style="width:100%;float:right;display:inline;">
									<span class="mbr-iconfont display-5" style="cursor: pointer;float:right;font-size:1rem" onclick="openNav('netoptsidenav','250px')">&#9776;
										<h6 class="mbr-iconfont display-5" style="font-size: 1rem; display: inline">Network Options</h6>
									</span>
									<div class="form-group" style="width: 175px; float: right; padding-right: 10px">
										<select class="form-control" id="whichnetwork" onchange="changeNetwork()">
											<option>GTEx Kinase Network</option>
											<option>TCGA Kinase Network</option>
											<option>ARCHS4 Kinase Network</option>
										</select>
									</div>
								</div>
							</div>
						</div> <!-- end coexpression tab -->
						<div class="tab-pane fade" id="pills-coreg" role="tabpanel" aria-labelledby="pills-coreg-tab">
							<svg id="coregsvg" width="960" height="600"></svg>
						</div>
					</div> <!-- end tab-contents div -->

				</div>
				
				<!-- Local Network Tab -->
				<div class="tab-pane border-top-0 rounded-bottom px-4 py-3 fade text-center" id="nav-local-network" role="tabpanel" aria-labelledby="nav-tables-tab">
					<svg id="coreg-network" height="500" width="1000"></svg>
				</div>
				
				<!-- Barchart Tab -->
				<div class="tab-pane border-top-0 rounded-bottom px-4 py-3 fade" id="nav-barchart" role="tabpanel" aria-labelledby="nav-tables-tab">
				</div>

				<!-- Clustergrammer Tab -->
				<div class="tab-pane border-top-0 rounded-bottom px-4 py-3 fade" id="nav-clustergram" role="tabpanel" aria-labelledby="nav-clustergram-tab">
					<!-- <img src="clustergrammer.png" class="w-100"> -->
					<iframe id="clustergram-iframe" class="w-100" frameborder="0" height="1000"></iframe> 
					<div id="clustergram-error" class="text-center py-5 display-5 d-none">
						Sorry, there has been an error generating the clustergrammer visualization.
						<br>
						Please <button type="button" class="btn btn-primary mx-0" onclick="generateClustergram();">click here</button> to try again.
					</div>
				</div>
			</div>
		</div>
	</section>


	<!-- Block 2: About ChEA3 -->
	<section class="mbr-section content4 cid-r0K1l83kKh mt-5" style="padding-bottom:0; padding-top:0" id="content4-o">


		<div class="container">
			<div class="media-container-row">
				<div class="title col-12 col-md-8">
					<h2 class="align-center pb-3 mbr-fonts-style display-2 my-3">About
						KEA3</h2>


				</div>
			</div>
		</div>
	</section>

	<section class="mbr-section article content12 cid-r0K1pQ9Oau" id="content12-q">


		<div class="container">
			<div class="media-container-row">
				<div class="mbr-text counter-container col-12 col-md-8 mbr-fonts-style display-7">
				Protein kinases are enzymes that catalyze the transfer of a phosphate group from ATP to another
protein on a threonine, a serine, or a tyrosine residue. The reversible addition of a phosphate group to a protein
can profoundly affect its reversible activity, stability, localization, and interactions with other molecules. Of the
hundreds of thousands of potential phosphorylation sites, each protein kinase recognizes between one to a few
hundred substrates. Thus, kinases serve a critical and central role in regulating essentially all cellular processes, 
and their aberrant activity is recognized as a cause of many human diseases. Phosphoproteomic signatures, which represent the differential phosphorylation state of
cellular proteins between two cellular states, can provide a snapshot of the intracellular signaling networks that
are differentially active between two conditions, for instance, between diseased and healthy cells. The
enrichment of known kinase substrates in a set of differentially phosphorylated proteins can serve as a marker
of the activity of the upstream kinases, thus providing insights into physiological and pathophysiological
mechanisms. Kinase Enrichment Analysis 3 (KEA3) infers upstream kinases
whose putative substrates are overrepresented in a user-inputted list of genes or differentially phosphorylated
proteins. The KEA3 database contains putative kinase-substrate interactions collected from publicly available datasets. Gene sets of putative kinase 
substrates are used as the primary units of analysis in KEA3. These gene sets are organized in gene set “libraries.” 
Libraries are supersets of kinase substrate sets that are aggregated based on the database from which they are derived.
					
					<br><br>
					
					<strong>Library Construction</strong><br> All protein and gene symbols were mapped to 2019 HGNC-approved gene symbols
					using the custom R package <a href = "https://github.com/MaayanLab/genesetr">genesetr</a>. Gene or protein symbols that could not be mapped using synonym or alias matching 
					were discarded. The set of 520 unique <a href = "https://www.genenames.org/">HGNC-mappable</a> human protein kinases used for KEA3 are derived from <a href = "https://www.ncbi.nlm.nih.gov/pubmed/12471243">Manning et al.</a>, 
					<a href = "https://www.ncbi.nlm.nih.gov/pubmed/17557329">Miranda-Saavedra and Barton</a>, and the <a href = "https://druggablegenome.net/">Illuminating the Druggable Genome</a> (IDG) project.
					<br><br>
					<i>Kinase-substrate interactions</i>
					<br>
					Kinase-substrated interactions were collected from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/#SD5" target="_blank">
					Cheng <i>et al.</i></a>, <a href = "https://omictools.com/phosd-tool">PhosD</a>, 
					and <a href="http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/" target="_blank">PTMsigDB</a> to generate 
					the <i>Cheng.KSIN</i>, <i>PhosD.All</i>, and <i>PTMsigDB</i> KEA3 libraries, respectively.
					The <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/#SD5" target="_blank">
					Cheng <i>et al.</i></a>kinase-substrate interaction network (KSIN) dataset incorporates human 
					kinase-substrate interactions from the <a href="">Phospho.ELM</a>, 
					<a href = "https://genome.cshlp.org/content/13/10/2363.full.html">HPRD</a>, 
					<a href = "https://phosphonetworks.org/">PhosphoNetworks</a>, and 
					<a href = "https://www.phosphosite.org/homeAction.action">PhosphoSitePlus</a> databases.
					<a href="http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/" target="_blank">PTMsigDB</a> contains a 
					collection of kinase phosphorylation signatures derived from <a href = "https://www.phosphosite.org/homeAction.action">PhosphoSitePlus</a>. 
					<a href = "https://omictools.com/phosd-tool">PhosD</a> predicts kinase-substrate interactions based on protein domains.
					
					<br><br>
					<i>Protein-protein interactions </i>
					<br>Kinase-interacting proteins were collected from the PPI databases <a href = "https://thebiogrid.org/">BioGRID</a>, 
					<a href = "https://mentha.uniroma2.it/">mentha</a>, <a href = "https://mint.bio.uniroma2.it/">MINT</a>, 
					<a href = "http://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie/">HIPPIE</a>, 
					<a href = "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/">Cheng <i>et al.</i></a>, 
					<a href = "http://honig.c2b2.columbia.edu/preppi">prePPI</a>, 
					and <a href = "https://string-db.org/">STRING</a> to generate the <i>BioGRID</i>, <i>mentha</i>, <i>MINT</i>, 
					<i>HIPPIE</i>, <i>Cheng.bind</i>, <i>prePPI</i>, and <i>STRING.bind</i> KEA3 libraries, respectively.
					The <a href = "https://thebiogrid.org/">BioGRID</a> and <a href = "https://mint.bio.uniroma2.it/">MINT</a>
					 databases contain PPIs from high- and low-throughput experiments that were manually curated 
					from the literature. <a href = "https://mentha.uniroma2.it/">Mentha</a> is a PPI network (PPIN) that contains molecular 
					interactions aggregated and updated weekly from <a href = "https://mint.bio.uniroma2.it/">MINT</a>, 
					<a href = "https://www.ebi.ac.uk/intact/">IntAct</a>, <a href = "https://thebiogrid.org/">BioGRID</a>, 
					<a href = "http://matrixdb.univ-lyon1.fr/">MatrixDB</a>, and the 
					<a href = "https://dip.doe-mbi.ucla.edu/dip/Main.cgi">Database of Interacting Proteins (DIP)</a>. 
					<a href = "http://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie/">HIPPIE</a> 
					aggregates experimentally determined PPIs 
					from <a href = "https://www.ebi.ac.uk/intact/">IntAct</a>,
					 <a href = "https://mint.bio.uniroma2.it/">MINT</a>, <a href = "https://thebiogrid.org/">BioGRID</a>, 
					 <a href = "https://genome.cshlp.org/content/13/10/2363.full.html">HPRD</a>, <a href = "https://dip.doe-mbi.ucla.edu/dip/Main.cgi">DIP</a>, <a href = "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC165503/">BIND</a>, and <a href = "http://mips.helmholtz-muenchen.de/proj/ppi/">MMPI</a>. 
					 <a href = "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/">Cheng <i>et al.</i></a> used PPIs collected from 
					 <a href = "https://www.ebi.ac.uk/intact/">IntAct</a>, <a href = "https://mint.bio.uniroma2.it/">MINT</a>, 
					 <a href = "https://thebiogrid.org/">BioGRID</a>, <a href = "https://dip.doe-mbi.ucla.edu/dip/Main.cgi">DIP</a>, 
					 <a href = "https://genome.cshlp.org/content/13/10/2363.full.html">HPRD</a>, 
					 and <a href = "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1347366/">MIPS MPact</a>.
					 <a href = "http://honig.c2b2.columbia.edu/preppi">PrePPI</a> consists of predicted PPIs that were determined using a 
					 Bayesian framework that relies on three-dimensional structural, functional evolutionary, 
					 and expression information to make PPI predictions.
					Note there is overlap and redundancy among the databases that aggregate 
					PPIs from the literature and from other PPI databases. 
					<br><br>
					<i>Other</i>
					<br>
					All kinase interactions in the <a href="https://string-db.org" target="_blank">STRING</a> database
					were used to generate the <i>STRING</i> KEA3 library. <a href="https://string-db.org" target="_blank">STRING</a> includes physical interaction, 
					co-expression, co-occurrence in the literature, 
					and evolutionary co-occurrence, among other association types.
					<br><br>
					<br>
					<strong>Kinase Substrate Over-representation Analysis</strong> -
					The goal of KEA3 is to predict kinases associated with user-input sets of
					genes/proteins.
					Discrete query gene/protein sets are compared to
					<a href="index.html#content4-13">KEA3 libraries</a> of putative kinase substrate sets assembled from multiple
					publically available datasets. The Fisher's Exact Test, with a background size of 20,000, is used to compare the 
					input gene/protein set to the kinase substrate
					sets in
					order to determine which kinases may be most closely associated with the input gene/protein set.
					<br>
					<br>

					<strong>Input</strong> - Users can submit a set of human gene or protein symbols for kinase enrichment analysis.
					Commonly, the gene set may be a set of
					differentially expressed genes, or a set of differentially phosphorylated proteins
					with the goal of forming hypotheses about which upstream kinases might be responsible for the changes in the phenotype.
					<br>
					<br>

					<strong>Kinase Co-expression Network Visualizations</strong> - The KEA3 application displays a scatter plot of points
					representing human kinases based on their co-expression similarity. This kinase coexpression network was constructed by
					conducting a <a
						href="https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/">Weighted Gene
						Co-expression Network Analysis (WGCNA)</a> over human kinase expression data. WGCNs were constructed from <a
						href="https://gtexportal.org/home/">GTEx</a> samples, <a href = "https://portal.gdc.cancer.gov">TCGA</a> samples, and 
						<a
						href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a> samples. 
					The network layouts were determined using Allegro Fruchterman-Reingold Clustering in <a
						href="https://cytoscape.org">Cytoscape</a>. For the <a
						href="https://gtexportal.org/home/">GTEx</a> networks, WGCNA kinase module summary expression profiles, also
					known as Eigen-genes,
					were correlated with gene expression profiles from all <a
						href="https://gtexportal.org/home/">GTEx</a> tissue samples to determine strong
					module-tissue associations. Each <a
						href="https://gtexportal.org/home/">GTEx</a> WGCN module may be colored by its most
					highly correlated tissue. Tissues are aggregated into more general tissue types (e.g. brain) and
					more specific tissue types (e.g. cerebellum, frontal cortex):

					<table class="table table-striped" id="gtextable"
						style="width: auto !important; margin-left: auto; margin-right: auto;">
						<thead>
							<tr>
								<th style="text-align: center; color:#767676"> General Tissue </th>
								<th style="text-align: center; color:#767676"> Specific Tissue </th>
							</tr>
						</thead>
						<tbody>
							<tr>
								<td style="text-align:left;"> Adipose Tissue </td>
								<td style="text-align:left;"> Adipose - Subcutaneous </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Adrenal Gland </td>
								<td style="text-align:left;"> Adrenal Gland </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Blood </td>
								<td style="text-align:left;"> Whole Blood </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Blood </td>
								<td style="text-align:left;"> Cells - EBV-transformed lymphocytes </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Blood Vessel </td>
								<td style="text-align:left;"> Artery - Tibial </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Brain </td>
								<td style="text-align:left;"> Brain - Cerebellum </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Brain </td>
								<td style="text-align:left;"> Brain - Frontal Cortex (BA9) </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Brain </td>
								<td style="text-align:left;"> Brain - Spinal cord (cervical c-1) </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Colon </td>
								<td style="text-align:left;"> Colon - Sigmoid </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Colon </td>
								<td style="text-align:left;"> Colon - Transverse </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Esophagus </td>
								<td style="text-align:left;"> Esophagus - Muscularis </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Esophagus </td>
								<td style="text-align:left;"> Esophagus - Mucosa </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Heart </td>
								<td style="text-align:left;"> Heart - Atrial Appendage </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Liver </td>
								<td style="text-align:left;"> Liver </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Muscle </td>
								<td style="text-align:left;"> Muscle - Skeletal </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Nerve </td>
								<td style="text-align:left;"> Nerve - Tibial </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Pancreas </td>
								<td style="text-align:left;"> Pancreas </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Pituitary </td>
								<td style="text-align:left;"> Pituitary </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Prostate </td>
								<td style="text-align:left;"> Prostate </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Skin </td>
								<td style="text-align:left;"> Skin - Sun Exposed (Lower leg) </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Skin </td>
								<td style="text-align:left;"> Cells - Transformed fibroblasts </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Testis </td>
								<td style="text-align:left;"> Testis </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Thyroid </td>
								<td style="text-align:left;"> Thyroid </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Uterus </td>
								<td style="text-align:left;"> Uterus </td>
							</tr>
						</tbody>
					</table>

					<a href = "https://portal.gdc.cancer.gov">TCGA</a> module Eigen-genes were correlated with tumor gene expression
					profiles to determine strong module-tumor type associations. Each <a href = "https://portal.gdc.cancer.gov">TCGA</a> WGCN module may be colored by its most highly correlated
					tumor type. Finally, the <a
						href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a> module Eigen-genes were correlated with <a
						href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a> tissue types. The <a
						href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a> network modules may be colored
					 by the most highly correlated tissue type.
					<br>
					<br>
				</div>
			</div>
		</div>
	</section>
	
	<!-- Tutorial SECTION -->
	
	<div class="container">
			<div class="media-container-row">
				<div class="title col-12 col-md-8">
					<h2 class="align-center pb-3 mbr-fonts-style display-2">
						Tutorial</h2>
					
					
				</div>
			</div>
		</div>
	

	<section class="mbr-section article content11 cid-r0K2ncv4vg" id="content4-about">


		<div class="container">
			<div class="media-container-row">
				<div class="mbr-text counter-container col-12 col-md-8 mbr-fonts-style display-7">
					<ol>
						<li><strong>Submit Your Query</strong> - Submit a query with one gene or protein symbol per row in the 
							text box, or upload a text file with one gene or protein symbol per line.
							KEA3 accepts HGNC-approved gene symbols (but is not case-sensitive) and will discard probe names, transcript IDs,
							and other unrecognizable IDs that have not been converted to gene symbols. You can
							use the example gene set available as a link above the submission box to gain
							familiarity with the expected format. After submitting your query, you will be presented
							with the results page. Most queries are processed in less than 30 seconds.
							<img src="./assets/chea3home/images/KEA3_submit.gif" style="width:100%">
							

						<li><strong>Navigating Your Results</strong> - Results tables from the integration methods MeanRank and TopRank 
						and from each KEA3 kinase substrate library are accessible from a dropdown menu.
						The MeanRank table is by default the first table shown because it performed the best across the KEA3 benchmarks. Tables may
							be searched by entering terms in the search box, sorted by column by clicking on the
							column header, and copied to the clipboard by clicking on the Copy button. Results
							for the KEA3 kinase substrate libraries are sorted by the Fisher's Exact Test p-value.
							This provides a ranking of kinases whose putative substrates
							are most closely similar to the query set. Integrated results, which take into account
							results from all libraries, are sorted in ascending order by score. Lower scores indicate
							more relevancy to the kinase substrate set. The top 100 results are returned for each table. The full kinase ranking and results
							for each integration method and library may be downloaded as a tab-separated (tsv) file using a link found at the footer of each
							results table. Full results are also available via the
							<a href="index.html#content4-z">KEA3 API</a>.
							<img src="./assets/chea3home/images/KEA3_tables.gif" style="width:100%"></li>


						<li><strong>Visualizing Your Results</strong> - A global kinase co-expression
							network, a local results-specific co-regulatory network, bar charts, and a clustergram are
							available to aid in interpreting the results.
							<ul>
							<br>
							  <li><strong>Kinase Co-expression Networks</strong></li>
							  
							  The kinase co-expression networks are provided to help users visualize their top ranking kinases
							in the context of the larger human kinase phosphorylation network. There are three networks generated from GTEx,
							TCGA, and ARCHS4 expression data. Users may toggle between these three networks using the dropdown menu. 
							<img
								src="./assets/chea3home/images/KEA3_coexp.gif" style="width:100%">
							
							
							The slider above the network designates the number of kinases to highlight in the network. The kinases that are highlighted are the top results
							from the library that is selected from library selection dropdown menu. 
							Network node coloring options provide additional information about the tissues or tumor types the kinases may be
							most active in. The network may be zoomed, panned, and saved as an SVG image. There are other options, for
							example, hiding the legend or switching to full screen. These options can be accessed by clicking
						    the "Network Options" hamburger.
								<br><br>
								 </li>
							  
							  <li><strong>Kinase Co-regulatory Networks</strong></li>
							  Kinase-kinase phosphorylation networks are dynamically generated using the top results of the selected library. Edges between kinases are defined 
							  by evidence from the KEA3 libraries and are directed where evidence from a KSI library supports the interaction and undirected where evidence from a PPI library supports the interaction. 
							  The supporting evidence for each
							  edge is summarized in a tooltip that appears upon hovering over each edge. The slider above the network indicates the number of top kinase results 
							  to include in the network from the library results indicated in the library selection dropdown.
							  <img
								src="./assets/chea3home/images/KEA3_coreg.gif" style="width:100%"> 
							  <li><strong>Bar Charts</strong></li>
							  Bar charts corresponding to the top kinases of the selected library display the -log10(pvalue) for KEA3 kinase substrate library results, and an integrated
							  rank score for the integrated library results. The numer of top kinases on the y-axis may be modifed using the slider.
							  							  <img
								src="./assets/chea3home/images/KEA3_bar.gif" style="width:100%"> 
							  
							  <li>Clustergram</li>
							  The interactive clustergram shows the overlapping query substrate targets among top library results.
							   <img
								src="./assets/chea3home/images/KEA3_cluster.gif" style="width:100%"> 
							</ul> 
							
					</ol>
					<br>
					<br>
					</div>
					</div>
					</div>
					</section>
				
	
	
	<!-- API SECTION -->
	<section class="mbr-section content4 cid-r0K2ViwrYH" style="padding-bottom:0" id="content4-z">


		<div class="container">
			<div class="media-container-row">
				<div class="title col-12 col-md-8">
					<h2 class="align-center pb-3 mbr-fonts-style display-2">API</h2>
					<h3 class="mbr-section-subtitle align-center mbr-light mbr-fonts-style display-5">
						KEA3 has an API to enable programmatic access for querying the KEA3 database</h3>

				</div>
			</div>
		</div>


		<section class="mbr-section article content1 cid-r0K4JJbbUf" id="content1-11" style="padding-bottom:0">


			<div class="container">
				<div class="media-container-row">
					<div class="mbr-text col-12 col-md-8 mbr-fonts-style display-7">
						The KEA3 REST API uses POST to transport user submitted JSON-formatted gene sets and JSON-formatted
						query results between the KEA3 server and the user’s script. The user gene set and optional
						additional information are encoded in JSON format.
						<br><br>
						<strong>Method:</strong> POST<br>
						<strong>URL:</strong> /KEA3/api/enrich/<br>
						<strong>Returns:</strong> JSON array of KEA3 library result objects<br>
						<strong>Parameters:</strong> <table style="display:inline-table;border-collapse:collapse;border-spacing:0" class="tg"><tr><th style="font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">query_name</th><th style="font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">String</th></tr><tr><td style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">gene_set</td><td style="font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:black;text-align:left;vertical-align:top">An array of strings</td></tr></table>
						<br><br>
						
					</div>
				</div>
			</div>
		</section>

		<section class="mbr-section article content11 cid-r0K4Kn3FRS" id="content11-12" style="padding-bottom:0px">


			<div class="container">
				<div class="media-container-row">
					<div class="mbr-text counter-container col-12 col-md-8 mbr-fonts-style display-7">
						<ol>
							<li><strong>Command-line Example</strong> - The following returns results from all KEA3
								libraries in JSON format:
								<pre
									style="width: 100%; overflow-x: scroll; white-space: nowrap">$ curl -d '{"query_name":"myQuery", "gene_set":["FOXM1","SMAD9","MYC","SMAD3","STAT1","STAT3"]}' -H 'Content-Type: application/json' https://amp.pharm.mssm.edu/KEA3/api/enrich/</pre>
								Redirect the JSON-formatted results to a file:
								<pre
									style="width: 100%; overflow-x: scroll; white-space: nowrap">$ curl -d '{"query_name":"myQuery", "gene_set":["FOXM1","SMAD9","MYC","SMAD3","STAT1","STAT3"]}' -H 'Content-Type: application/json' https://amp.pharm.mssm.edu/KEA3/api/enrich/ > results.json</pre>
								</a></li>
							<!-- <li><strong>Python Example</strong> - 
								<pre
								style="width: 100%; overflow-x: scroll; white-space: nowrap"></pre></li> -->
							<li><strong>R Example</strong> -
								<pre style="width: 100%; overflow-x: scroll">
									rm(list = ls())
									library(httr)
									library(jsonlite)
									
									genes = c("SMAD9","FOXO1","MYC","STAT1",'STAT3',"SMAD3")
									
									url = "https://amp.pharm.mssm.edu/KEA3/api/enrich/"
									encode = "json"
									payload = list(query_name = "myQuery", gene_set = genes)
									
									#POST to KEA3 server
									response = POST(url = url, body = payload, encode = encode)
									json = content(response, "text")
									
									#results as list of R dataframes
									results = fromJSON(json)
								
								</pre>
							</li>
						</ol>
					</div>
				</div>
			</div>
		</section>
	</section>

	<!-- DOWNLOAD LIBRARIES SECTION -->

	<section class="mbr-section content4 cid-r0K4Mm8LAA" style="padding-bottom:0" id="content4-13">


		<div class="container align-center">
			<div class="media-container-row">
				<div class="title col-12 col-md-8">
					<h2 class="align-center pb-3 mbr-fonts-style display-2">
						Download</h2>


				</div>
				
			</div>
			"Primary" libraries are used by KEA3 to conduct kinase enrichment analysis. "Benchmarking" libraries were used to benchmark KEA3. "Additional" libraries are available as an additional resource to the community. "Primary" and "Additional" libraries used in KEA3 were generated only include kinases with four or more interactions, and all gene symbols are HGNC approved.
		</div>
		<br>
	</section>
	
	<div class="container">
			<div class="media-container-row">
	
	<head><title> </title></head>
  
  <body>
  <table border="1" class="dataframe mystyle">
  <thead>
    <tr style="text-align: initial; text-align: center; color:#767676">
      <th>Library</th>
      <th>Library Type</th>
      <th>Organism</th>
      <th>Unique Kinases</th>
      <th>Interactions</th>
      <th>Unique Substrates</th>
      <th>Phosphosites?</th>
      <th>Download</th>
    </tr>
  </thead>
  <tbody>
  
  <!-- Primary Libraries -->
  
   <tr>
      <td><a style="text-decoration:none" href="https://downloads.thebiogrid.org/" target="_blank">BioGRID</a></td>
      <td>Primary</td>
      <td>Human</td>
      <td>256</td>
      <td>18722</td>
      <td>6505</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_biogrid_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
    <tr>
      <td><a style="text-decoration:none" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116514/#SD5" target="_blank">Cheng</a></td>
      <td>Primary</td>
      <td>Human</td>
      <td>311</td>
      <td>12117</td>
      <td>3738</td>
      <td>No</td>
	  <td><a href="assets/tflibs/output_cheng_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
     <tr>
      <td><a style="text-decoration:none" href="http://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie/" target="_blank">HIPPIE</a></td>
      <td>Primary</td>
      <td>Human</td>
      <td>271</td>
      <td>25578</td>
      <td>7336</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_hippie_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
     <tr>
      <td><a style="text-decoration:none" href="http://mentha.uniroma2.it/index.php" target="_blank">Mentha</a></td>
      <td>Primary</td>
      <td>Human</td>
      <td>259</td>
      <td>18761</td>
      <td>6578</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_mentha_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
     <tr>
      <td><a style="text-decoration:none" href="https://mint.bio.uniroma2.it/" target="_blank">MINT</a></td>
      <td>Primary</td>
      <td>Human</td>
      <td>74</td>
      <td>1253</td>
      <td>704</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_mint_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
     <tr>
      <td><a style="text-decoration:none" href="http://www.phosphonetworks.org/index.html" target="_blank">PhosphoNetworks (literature)</a></td>
      <td>Primary</td>
      <td>Human</td>
      <td>158</td>
      <td>3379</td>
      <td>943</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_phosphonetworks-comKSI_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
     <tr>
      <td><a style="text-decoration:none" href="http://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/" target="_blank">PTMsigDB</a></td>
      <td>Primary</td>
      <td>Human</td>
      <td>140</td>
      <td>4155</td>
      <td>1892</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_ptmsigdb_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
     <tr>
      <td><a style="text-decoration:none" href="https://string-db.org/cgi/input.pl?sessionId=cOMzocvk3ztX&input_page_show_search=on" target="_blank">String</a></td>
      <td>Primary</td>
      <td>Human</td>
      <td>281</td>
      <td>321383</td>
      <td>16404</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_string_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
     <!-- Benchmarking Libraries -->
    
    <tr>
      <td><a style="text-decoration:none" href="https://amp.pharm.mssm.edu/Enrichr/#stats" target="_blank">Up-regulated kinase perturbations from GEO</a></td>
      <td>Benchmarking</td>
      <td>Human, Mouse, Rat</td>
      <td>90</td>
      <td>94657</td>
      <td>20511</td>
      <td>No</td>
	  <td><a href="assets/tflibs/single_kinase_perts_from_GEO_dn.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
    <tr>
      <td><a style="text-decoration:none" href="https://amp.pharm.mssm.edu/Enrichr/#stats" target="_blank">Down-regulated kinase perturbations from GEO</a></td>
      <td>Benchmarking</td>
      <td>Human, Mouse, Rat</td>
      <td>90</td>
      <td>94657</td>
      <td>20511</td>
      <td>No</td>
	  <td><a href="assets/tflibs/single_kinase_perts_from_GEO_up.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
    <tr>
      <td><a style="text-decoration:none" >Up- and down-regulated kinase perturbations from GEO</a></td>
      <td>Benchmarking</td>
      <td>Human, Mouse, Rat</td>
      <td>90</td>
      <td>189314</td>
      <td>20555</td>
      <td>No</td>
	  <td><a href="assets/tflibs/single_kinase_perts_from_GEO_updn.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
    <!-- Additional Libraries -->
    
    <tr>
      <td><a style="text-decoration:none" href="http://mizuguchilab.org/PSOPIA/theory.html" target="_blank">Psopia</a></td>
      <td>Additional</td>
      <td>Human</td>
      <td>38</td>
      <td>522</td>
      <td>397</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_psopia_fourplusinteractions_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
    <tr>
      <td><a style="text-decoration:none" href="http://mentha.uniroma2.it/index.php" target="_blank">Mentha</a></td>
      <td>Additional</td>
      <td>Human</td>
      <td>304</td>
      <td>23702</td>
      <td>7673</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_mentha_fourplusinteractions_mouse_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
     <tr>
      <td><a style="text-decoration:none" href="https://downloads.thebiogrid.org/" target="_blank">BioGRID</a></td>
      <td>Additional</td>
      <td>Mouse</td>
      <td>256</td>
      <td>18722</td>
      <td>6505</td>
      <td>No</td>
      <td><a href="assets/tflibs/output_biogrid_mouse_human_mapped.gmt" download><img src="assets/images/download_icon.png" height="17" width="17"></a></td>
    </tr>
    
    
    
  </tbody>
</table>



	
	</div></div>
	

	<!--Footer-->
	<br><br>
	<div class="row px-5 justify-content-center">
		<!-- Affiliations Column -->
		<div class="col-md-5 col-sm small">
			<h5 class="mb-3">Affiliations</h5>
			<div><a class="black" href="http://icahn.mssm.edu/research/labs/maayan-laboratory" target="_blank">The
					Ma'ayan Lab</a></div>
			<div><a class="black" href="http://www.lincs-dcic.org/" target="_blank">BD2K-LINCS Data
					Coordination and Integration Center (DCIC)</a></div>
			<div><a class="black" href="http://www.lincsproject.org/">NIH LINCS program</a></div>
			<div><a class="black" href="https://nihdatacommons.us/" target="_blank">Data Commons Pilot
					Project
					Consortium (DCPPC)</a></div>
			<div><a class="black" href="https://druggablegenome.net/KMC_ISMMS" target="_blank">NIH
					Illuminating the Druggable Genome (IDG) Program</a></div>
			<div><a class="black" href="http://icahn.mssm.edu/" target="_blank">Icahn School of Medicine at
					Mount Sinai</a></div>
			<div><a class="black" href="http://icahn.mssm.edu/research/bioinformatics" target="_blank">Mount
					Sinai Center for Bioinformatics</a></div>
		</div>

		<!-- Share Column -->
		<div class="col-md-2 col-sm small">
			<h5 class="mb-3">License</h5>
			<a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">
				<img alt="Creative Commons License" style="border-width:0;"
					src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png">
			</a>
			<div class="micro mt-2">
				<p>All contents on this site are covered by the <a rel="license"
						href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative
						Commons Attribution 4.0 International License</a>.</p>
			</div>
		</div>

		<!-- Logo Column -->
		<div class="col-md-3 col-sm px-5 mt-3">
			<a href="http://icahn.mssm.edu/research/bioinformatics" target="_blank">
				<img src="https://amp.pharm.mssm.edu/archs4/images/mountsinai.png" class="w-100 px-3">
			</a>
		</div>
	</div>

	<iframe id="dwnld_iframe" style="display: none;"></iframe>
	
	<script type="text/javascript">
		var host = "http://localhost:8080/";
	</script>
	<script src="assets/web/assets/jquery/jquery-3.3.1.min.js" type="text/javascript"></script>
	<script src="assets/popper/popper.min.js" type="text/javascript"></script>
	<!-- 	bootstrap v4  -->
	<script src="assets/bootstrap/js/bootstrap.min.js" type="text/javascript"></script>
	<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js" type="text/javascript"></script>

	<!--a bunch of d3 scripts-->
	<script src="assets/d3/d3.v4.min.js" type="text/javascript"></script>
	<!-- <script src="assets/d3/d3.v5.min.js"></script> -->
	<script src="https://d3js.org/d3-selection-multi.v1.js"></script>
	<script src="https://d3js.org/d3-scale-chromatic.v1.min.js"></script>

	<script src="https://cdn.jsdelivr.net/npm/bootstrap-select@1.13.9/dist/js/bootstrap-select.min.js"></script>


	<!--bootstrap/mobirise--->

	<script src="assets/tether/tether.min.js" type="text/javascript"></script>

	<script src="assets/smoothscroll/smooth-scroll.js" type="text/javascript"></script>
	<script src="assets/touchswipe/jquery.touch-swipe.min.js" type="text/javascript"></script>
	<script src="assets/datatables/jquery.dataTables.min.js" type="text/javascript"></script>
	<script src="assets/datatables/dataTables.buttons.min.js" type="text/javascript"></script>
	<script src="assets/datatables/jszip.min.js" type="text/javascript"></script>
	<script src="assets/datatables/buttons.html5.min.js" type="text/javascript"></script>
	<script src="assets/datatables/pdfmake.min.js" type="text/javascript"></script>
	<script src="assets/datatables/vfs_fonts.js" type="text/javascript"></script>
	<script src="assets/mbr-switch-arrow/mbr-switch-arrow.js" type="text/javascript"></script>
	<script src="assets/sociallikes/social-likes.js" type="text/javascript"></script>
	<script src="assets/dropdown/js/script.min.js" type="text/javascript"></script>
	<script src="assets/theme/js/script.js" type="text/javascript"></script>
	<script src="assets/chea-query/download_results.js" type="text/javascript"></script>

	<script src="assets/datatables/dataTables.bootstrap4.min.js" type="text/javascript"></script>
	<script src="assets/datatables/buttons.colVis.min.js" type="text/javascript"></script>
	<script src="assets/spectrum/spectrum.js" type="text/javascript"></script>
	<script src="https://d3js.org/d3-array.v1.min.js" type="text/javascript"></script>

	<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.8.0/Chart.bundle.min.js"
		type="text/javascript"></script>
	<!-- <script src="assets/hitcounter.js"></script> -->



	<!-- OpenCPU scripts -->
	<script src="assets/chea-query/generateCoregNetwork.js" type="text/javascipt"></script>
	<script src="assets/chea-query/color-array.js" type="text/javascript"></script>
	<script src="assets/chea-query/plot_results.js" type="text/javascript"></script>
	<script src="assets/chea-query/generate_clustergram.js" type="text/javascript"></script>
	<script src="assets/chea-query/generate_coreg_network.js" type="text/javascript"></script>
	<script src="assets/chea-query/submit-query2.js" type="text/javascript"></script>
	<script src="assets/networkd3/tissue_legends.js" type="text/javascript"></script>
	<script src="assets/networkd3/network3.js" type="text/javascript"></script>
	<script src="assets/chea-query/generateCoregNetwork.js" type="text/javascript"></script>
	<script src="assets/chea3home/chea3homepage.js" type="text/javascript"></script>

	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-6277639-43"></script>
	<script>
		window.dataLayer = window.dataLayer || [];
		function gtag() { dataLayer.push(arguments); }
		gtag('js', new Date());

		gtag('config', 'UA-6277639-43');
		$('[data-toggle="popover"]').popover();

	</script>
	<script type="text/javascript">if ((location.protocol != 'https:') && (location.hostname != 'localhost')){location.href = 'https:' + window.location.href.substring(window.location.protocol.length);}</script>




</body>

</html>